services/compute/virtual-machines/taxonomy.md

```diff
@@ -0,0 +1,47 @@
+# Virtual Machines Taxonomy
+
+This _service-level taxonomy_ documents the minimal set of features
+that should be present for a service to be considered portable for
+use in financial services ecosystems.
+
+## Taxonomy
+
+| Taxonomy ID | Feature | Description |
+| ----------- | ------- | ----------- |
+|	CCC-030101	|	Instance Types	|	Offers a range of instance types with different specifications for CPU, GPU, memory, storage optimisation and networking capacity.	|
+|	CCC-030102	|	Operating System Options 	|	A selection of operating systems for virtual machine instances.	|
```

Comment:
Same as my comment on CCC-030101

Reply:
Just checking, do you mean you I could list the potential OSs as separate features? If so, how should we determine which ones are required?

Reply:
I think that listing out available options is just asking for trouble! Perpetual updates required... for ever! I think stating that multiple OSes *could* or *are* supported (without naming them) should suffice.